NEXT_VERSION.md

```diff
@@ -22,6 +22,7 @@ Changes since the last version (oldest first):
 * (patch) #256 - bug in 247, defaults override config (akamyshanov)
 * (patch) #258 - threadsafe SessionState.Get/Set to fix garbled message issue (jacsuper)
 * (patch) #251 - restore the session type check in AbstractInitiator.Start (oract)
+* (patch) #255 - release ClientHandlerThread resources on client disconnect (mgatny)
```

Comment:
I specifically didn't do that because putting them at the bottom is guaranteed to cause merge conflicts with everyone else who does the same, requiring non-auto merge.